src/com/company/Main.java

```diff
@@ -133,113 +172,188 @@ public static void _readBookList() {
-                        if (book != null && book.availableCount > 0) { // checking corner cases such as book is not in library or the book is out of stock
-                            book.availableCount--;
-                            book.checkoutCounter++;
-                        }
-                        else { //handling error case when we try to check out a book that is out of stock
-                            System.out.println("Invalid Book ID found, No such book in Library");
-                        }
-                }
-                else if(record[0].equalsIgnoreCase("checkIn")) {  //here when the book is checked in then we simply increase the available counter by 1 , checkout counter remains same
-                    book = library.findBook(Integer.parseInt((record[1].trim()))); // taking care of any strings that might have space in it before parsing to integer
-                        if (book != null) { // taking care if the book is not present in library
-                            book.availableCount++;
-                        }
-                        else { // suitable message to denote that the book we're trying to cehck in was never present in library
-                            System.out.println("Invalid Book ID found, No such book in Library");
-                        }
-                }
-                else {  // finaly taking care of any invalid entried in promptsPS4.txt file
+                if (inOut.equalsIgnoreCase("checkOut")) { //if book status is 'checkout' then we reduce the available
+                    // count by 1 and increase the checkout counter by 1
+                    book = library.findBook(library.rootBook, bkID);
+                    if (book != null && book.availableCount > 0) { // checking corner cases such as book is not in library or the book is out of stock
+                        book.availableCount--;
+                        book.checkoutCounter++;
+                    } else { //handling error case when we try to check out a book that is out of stock
+                        System.out.println("Invalid Book ID found, No such book in Library");
+                    }
+                } else if (inOut.equalsIgnoreCase("checkIn")) {  //here when the book is checked in then we simply
+                    // increase the available counter by 1 , checkout counter remains same
+                    book = library.findBook(library.rootBook, bkID);
+                    if (book != null) { // taking care if the book is not present in library
+                        book.availableCount++;
+                    } else { // suitable message to denote that the book we're trying to cehck in was never present in library
+                        System.out.println("Invalid Book ID found, No such book in Library");
+                    }
+                } else { // i don't think this else is required anymore coz we already checked but still
```

Comment:
yes it can be removed only after verifying that all edge cases of wrong spelling and incorrect format is handled in trigger for prompts